### DISCUSSION TITLE
Add logo to the  config portal

### DISCUSSION THREAD

**Discussion by bingran** [khoih-prog/WiFiManager_RTL8720#6] [Q&A]
I am trying to add  a logo on top of the on the config portal . 
But i notice when the string is too long it unable to display any idea ?

**Comment by jonogan** [comment]
I'm seeing similar results. I've converted a small image to base64 and tried to include it via css background style:
`.logo{width: 290px;height: 63px;background: url(data:image/png;base64,<base64 string>)}`
and also in a plain image tag added to the .h file in the html section starting on line 190. In chrome, I get a ERR_CONTENT_LENGTH_MISMATCH error and can't seem to load the page. 

But if I take the generated html from the serial monitor and load it as a file in the browser independently, it loads fine, with the logo intact. Any advice here for us?

**Reply by bingran** [reply]
Hi jonogan , i think some of your changes cause the ERR_CONTENT_LENGTH_MISMATCH , i manage to display the html with a web page around 3k byte actually but not more than that.
